CHANGELOG.md

```diff
@@ -41,7 +41,7 @@ BUG FIXES:
 
 * agent: (Consul Enterprise) Fixed an issue where the snapshot agent's HTTP client config was being ignored in favor of the HTTP command-line flags.
```

Comment:
Given it is a historical record I recommend we drop changes to `CHANGELOG.md`, less of a living document.